### DISCUSSION TITLE
Search/filter SQL join query

### DISCUSSION THREAD

**Discussion by KristofferBroberg** [Budibase/budibase#3695] [Help] [answered]
Hi all!

Following the docs, I've still not managed to create search functionality for my table.

So, basically SELECT * FROM table1 LEFT JOIN table2 ON table1.id = table2.id; is run to createfed to table. To this I´d like to add a search box that searches/filters based on id.

Which is the correct way of doing this? The docs concerning search and filter won't fit this scenario? If not, why? Should I create a parameter for the query and somehow use this to create search front end?

Appreciate if someone could point me in the right direction!

**Comment by KristofferBroberg** [comment, marked as answer]
All of solved with november release. I'll mark this as answered, because it seems I can'r remove my question altogether.

**Reply by Pierrelai99** [reply]
hey bro can you teach me how to use parameter to filter? because I cannot do this

**Reply by mjashanks** [reply]
@Pierrelai99 happy to answer your question if you raise a new discussion and put more detail behind it :)
